apps/synapse-oauth-signin/index.html

```diff
@@ -48,19 +48,16 @@
     <script src="https://unpkg.com/markdown-it-inline-comments@1.0.1/dist/markdown-it-inline-comments.min.js"></script>
     <script src="https://unpkg.com/markdown-it-br@1.0.0/dist/markdown-it-br.min.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/markdown-it-container@2.0.0/dist/markdown-it-container.min.js"></script>
-    <script
-      async
-      src="https://www.googletagmanager.com/gtag/js?id=UA-29804340-1"
-    ></script>
     <title>Sage Bionetworks Sign-in</title>
-    <script>
-      window.dataLayer = window.dataLayer || []
-      function gtag() {
-        dataLayer.push(arguments)
-      }
-      gtag('js', new Date())
-      gtag('config', 'UA-29804340-1')
-    </script>
+      <!-- Google Tag Manager -->
+      <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        })(window,document,'script','dataLayer','GTM-KPW4KS62');
```

Comment:
Is it ok that the IDs are the same (GTM-KPW4KS62) ?

I dunno if that's the account ID or if it should be unique to each property

Reply:
We're currently sending all data to the same GA stream (and now the same GTM "container").  Historically we break apart by domain later.  We could certainly organize it differently (different GA property for each website), but this would be additional overhead (especially for each of the portal websites).  So this change is in line with how it has historically been done, for better or worse.